controller/src/main/java/io/pravega/controller/store/stream/tables/State.java

```diff
@@ -16,5 +16,8 @@
     UNKNOWN,
     CREATING,
     ACTIVE,
+    UPDATING,
```

Comment:
What is the reason for additional state `UPDATING`. Since, we are queuing up all scale requests now, at any point of time at most one scale operation can be ongoing, isn't it?